src/main/js/main_html5.js

```diff
@@ -1195,6 +1195,11 @@ require("../../../html5-common/js/utils/environment.js");
                       _video.currentTime);
           _.defer(raiseEndedEvent);
         }
+        if(OO.isSafari && !OO.isIos && !_video.ended && Math.round(_video.currentTime) === Math.round(_video.duration))
+        {
+          this.controller.notify(this.controller.EVENTS.SEEKED);
```

Comment:
Re EVENTS.SEEKED, when I seek to the very end or just past it, I intermittently fail to or manage to see this event get raised. Thus far unpredictable.